Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1422,6 +1422,136 @@ def condition(self, fragment, stepList, key, hasHarvest):
     offset = 0.21,
 )
 
+class UpgradeWorkflow_ProdLikeRunningPU(UpgradeWorkflow_ProdLike):
+    def __init__(self, suffix, offset, fixedPU,
+         steps = [
+        'GenSimHLBeamSpot14',
```

Comment:
Why do we need all of these steps in `steps`? We don't need to treat NoPU in this customization.

Reply:
Indeed you are right I will remove them.